Nadim.CinemaReservationSystem.Web/ClientApp/src/components/Header/NavPanel.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Route, Link } from 'react-router-dom';
+import { Button, FormControl} from 'react-bootstrap';
+
+const NavPanel = (props) =>{
+    if (props.role !== 'admin'){
+        return(
+            <Route path="/sessions" render={() => (
+                <React.Fragment>
+                    <div className="header-column">
+                        <Link to="/">
+                            <div className="logo">
+                                <img
+                                    src={require('../Images/cinema_logo_white.png')}
+                                    alt="logo"
+                                />
+                            </div>
+                        </Link>
+                    </div>
+                    <div className="header-column">
+                        <div className="search-container">
+                            <img 
```

Comment:
You shouldn't use img tag here as this is decorative image. You might want to try using some icon library.